cetmix_tower_server/tests/test_reference_mixin.py

```diff
@@ -92,3 +92,109 @@ def test_search_by_reference(self):
         # Search using malformed (case sensitive)
         search_result = self.ServerTemplate.get_by_reference("not_much_template")
         self.assertEqual(len(search_result), 0, "Result should be empty")
+
+    def test_prepare_references_valid_input(self):
```

Comment:
Which function exactly is tested here? Opt for using `test_<function_name>_..` naming pattern for better readability.

Reply:
Thanks, I'll fix it